README.md

```diff
@@ -190,7 +190,8 @@ make power-<on,reset,off>-<machine name>
 
 There are two versions, or flavors, of the mini-lab environment which differ in regards to the NOS running on the leaves:
 
-- `sonic`: runs 2 SONiC switches
+- `sonic`: runs two SONiC switches
```

Comment:
The above sentence should be adjusted. "There are **four** version, or flavors...".